src/Workspaces/SharedUtilitiesAndExtensions/Compiler/VisualBasic/Services/SyntaxFacts/VisualBasicSyntaxFacts.vb

```diff
@@ -2207,13 +2207,13 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.LanguageServices
                     Return DeclarationKind.Namespace
                 Case SyntaxKind.ImportsStatement
                     Return DeclarationKind.NamespaceImport
-                Case SyntaxKind.ClassBlock
+                Case SyntaxKind.ClassBlock, SyntaxKind.ClassStatement
```

Comment:
Currently there is no special logic for things other than class, so only this change seems to have a noticeable and testable behavioral change. But I fixed all others.